hooks/use-webrtc.ts

```
@@ -0,0 +1,394 @@
+      console.log('Data channel closed')
+    }
+
+    return dc
+  }, [audioConfig, handleError])
+
+  const handleRealtimeMessage = useCallback((message: any) => {
+    console.log('Received message:', message.type)
+
+    switch (message.type) {
+      case 'conversation.item.input_audio_transcription.completed':
+        const transcript = message.transcript
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)